mizar/obj/endpoint.py

```diff
@@ -24,6 +24,7 @@
 from mizar.common.rpc import TrnRpc
 from mizar.common.constants import *
 from mizar.common.common import *
+from mizar.obj.data_networkpolicy import *
```

Comment:
Why are you calling another obj class from on obj class? obj should be independent from each other. If you need add network policy to endpoint, this should be done in the operator.

Reply:
Removed